mmcv/engine/test.py

```diff
@@ -4,15 +4,17 @@
 import shutil
 import tempfile
 import time
+from typing import Optional, Union
 
 import torch
 import torch.distributed as dist
+import torch.nn as nn
 
 import mmcv
 from mmcv.runner import get_dist_info
 
 
-def single_gpu_test(model, data_loader):
+def single_gpu_test(model: nn.Module, data_loader: nn.Dataloader) -> list:
```

Comment:
```suggestion
def single_gpu_test(model: nn.Module, data_loader: DataLoader) -> list:
```

Reply:
Need to fix the type of `data_loader` in docstring.

https://github.com/open-mmlab/mmcv/blob/569d588e155159fe8b9c5d9b08469a2814dd7e35/mmcv/engine/test.py#L55

```python
data_loader (Dataloader): Pytorch data loader. 
```